citadel/tutorials.md

```diff
@@ -15,13 +17,10 @@ These tutorials cover general concepts to help get you started with Ignition.
 * [Model Insertion from Fuel](fuel_insert)
 * [Keyboard Shortcuts](hotkeys)
 
-## Basics tutorials
+## ROS integration
 
-* [Building Your Own Robot](building_robot)
-* [Moving the Robot](moving_robot)
-* [SDF Worlds](sdf_worlds)
-* [Sensors](sensors)
-* [Actors](actors)
+* [ROS Integration](ros_integration)
```

Comment:
Is this a new tutorial that's going to be created?

Reply:
Oh never mind, it only exists for C and F. I was clicking around on the website and not finding it for G.